warden/x/gmp/types/abi_types.go

```diff
@@ -0,0 +1,9 @@
+package types
+
+import "github.com/ethereum/go-ethereum/accounts/abi"
+
+// These are the types we use to encode and decode data to and from the GMP.
+var (
+	destinationContractAddressType, _  = abi.NewType("address", "address", nil)
+	destinationContractCalldataType, _ = abi.NewType("bytes", "bytes", nil)
+)
```

Comment:
in these cases I like to panic on errors (you never know if the go-ethereum has a breaking change or whatever, you end up with unexpected behaviours)

if you want you can do something like this:

```suggestion
// These are the types we use to encode and decode data to and from the GMP.
var (
	destinationContractAddressType  = must(abi.NewType("address", "address", nil))
	destinationContractCalldataType = must(abi.NewType("bytes", "bytes", nil))
)

func must[T any](t T, err error) T {
	if err != nil {
		panic(err)
	}
	return t
}
```